src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dq/builders/DQGeneratorFormBuilder.java

```diff
@@ -203,12 +224,14 @@ private String createStatementOfTruthText(boolean respondentState) {
         String statementOfTruth = role.equals(DEFENDANT)
             ? "The defendant believes that the facts stated in the response are true."
             : "The claimant believes that the facts in this claim are true.";
-        statementOfTruth += String.format("\n\n\nI am duly authorised by the %s to sign this statement.\n\n"
-                                              + "The %s understands that the proceedings for contempt of court "
-                                              + "may be brought against anyone who makes, or causes to be made, "
-                                              + "a false statement in a document verified by a statement of truth "
-                                              + "without an honest belief in its truth.",
-                                          IntStream.range(0, 2).mapToObj(i -> role).toArray());
+        statementOfTruth += String.format(
+            "\n\n\nI am duly authorised by the %s to sign this statement.\n\n"
+                + "The %s understands that the proceedings for contempt of court "
+                + "may be brought against anyone who makes, or causes to be made, "
+                + "a false statement in a document verified by a statement of truth "
+                + "without an honest belief in its truth.",
+            IntStream.range(0, 2).mapToObj(i -> role).toArray()
```

Comment:
is it not easy to role, role instead of using this IntStream looks like overcomplicating

Reply:
this is just showing as a change due to code formatting.